subgraphs/uniswap-forks-swap/protocols/pancakeswap-v2-swap/config/deployments/pancakeswap-v2-swap-bsc/configurations.json

```diff
@@ -5,7 +5,7 @@
     "address": "0xca143ce32fe78f1f7019d7d551a6402fc5350c73",
     "startBlock": 6809737
   },
-  "graftEnabled": false,
-  "subgraphId": "",
-  "graftStartBlock": 0
+  "graftEnabled": true,
```

Comment:
turn to false to merge into master.

in my mind we should not merge grafts in bc they may or may not exist in the future and if we want to graft it should be deliberate (not by accident)